src/test/java/teammates/test/cases/ui/browsertests/AdminEmailPageUiTest.java

```diff
@@ -66,22 +79,169 @@ private void testCompose() {
         emailPage.inputSubject("Email Subject");
         emailPage.inputContent("Email to save");
         emailPage.clickSaveButton();
-        emailPage.verifyHtmlMainContent("/adminEmailSaveSuccess.html");
+        assertTrue(hasStatusMessageSaveSuccess());
     }
 
     private void testSent() {
         emailPage.clickSentTab();
-        emailPage.verifyHtmlMainContent("/adminEmailSentPage.html");
+        assertTrue(isEmailSentDataDisplayCorrect());
     }
 
     private void testDraft() {
         emailPage.clickDraftTab();
-        emailPage.verifyHtmlMainContent("/adminEmailDraftPage.html");
+        assertTrue(isEmailDraftDataDisplayCorrect());
     }
 
     private void testTrash() {
         emailPage.clickTrashTab();
-        emailPage.verifyHtmlMainContent("/adminEmailTrashPage.html");
+        assertTrue(isEmailTrashDataDisplayCorrect());
+    }
+    
+    private boolean isEmailComposePageCorrect() {
+        return emailPage.isElementPresent(By.id("addressReceiverEmails"))
+            && emailPage.isElementPresent(By.name("emailsubject"))
+            && emailPage.isElementPresent(By.className("mce-tinymce"))
+            && emailPage.isElementPresent(By.id("adminEmailBox"))
+            && emailPage.isElementPresent(By.id("composeSubmitButton"))
+            && emailPage.isElementPresent(By.id("composeSaveButton"));
+    }
+    
+    private boolean hasStatusMessageNoRecipient() {
+        return emailPage.getStatus().equals("Error : No reciver address or file given");
+    }
+    
+    private boolean hasStatusMessageRecipientEmailFormatError() {
+        return emailPage.getStatus().contains("\"recipient\" is not acceptable to TEAMMATES as an email "
```

Comment:
Maybe make the `recipient` a variable?
The `emailPage.getStatus().contains()` part can also be extracted into another method to reduce code duplication.

Honestly, these methods seem pretty pointless.
If you want to focus on readability, it may be better to have different methods for each type of test (`___TS()`) instead of creating different methods for every string comparison.